cross/git/PLIST

```diff
@@ -6,6 +6,16 @@ lnk:bin/git-upload-archive
 lnk:bin/git-upload-pack
 bin:bin/scalar
 rsc:etc/gitconfig
+bin:libexec/git-core/git
+bin:libexec/git-core/git-daemon
+bin:libexec/git-core/git-http-fetch
+bin:libexec/git-core/git-http-push
+bin:libexec/git-core/git-imap-send
+bin:libexec/git-core/git-remote-http
+bin:libexec/git-core/git-sh-i18n--envsubst
+bin:libexec/git-core/git-shell
+bin:libexec/git-core/scalar
 rsc:libexec/git-core/
 rsc:share/git-core/
+rsc:share/gitweb/
```

Comment:
@Fisiu is there any benefit of including gitweb in the package?

We intentionally removed this in #5035. It was required by our git-server package that is not maintained for a long time.
As we have a gitea package and Synology provides a git-server package, there is no reason to keep this piece of code in the package.

Reply:
I copied one line to much from generated PLIST. Should be good now.